nibabies/workflows/anatomical/base.py

```diff
@@ -612,6 +612,9 @@ def init_infant_single_anat_wf(
             "This workflow uses only T1w or T2w inputs, but both contrasts are available."
         )
 
```

Comment:
FYI - I decided to place the check here, because that is where you seem to be checking for the T[1,2]w images already.